authors/apps/comments/migrations/0001_initial.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.1.7 on 2019-04-17 08:13
+
+from django.db import migrations, models
+import django.db.models.deletion
+import mptt.fields
```

Comment:
I hope you didn't manually add this, It may cause more issues touching the migrations file.

Reply:
No, this was created using the `python manage.py makemigrations` command.